services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_utils.py

```diff
@@ -239,3 +242,29 @@ async def _pull_image_with_progress(
             await progress_cb(total_current, total_total)
 
         await log_cb(f"pulling {shorter_image_name}: {pull_progress}...")
+
+
+@cached()
+async def supports_volumes_with_quota() -> bool:
+    async with docker_client() as docker:
+        docker_volume: Optional[DockerVolume] = None
+        volume_name = f"check-quota-{uuid4()}"
+        try:
+            docker_volume = await docker.volumes.create(
+                {
+                    "name": volume_name,
+                    "Driver": "local",
+                    "DriverOpts": {"size": "1m"},
+                }
+            )
+        except aiodocker.DockerError as e:
+            if (
+                f"create {volume_name}: quota size requested but no quota support"
```

Comment:
Messages to identify an error is dangerous since those are for humans and might change. What about status code or any other error code they might provide?

- Can you pls add the link to where is this message documented? This way we can check if the message changed?
- Can you add this error in a constant and use it in the tests as well ?  ``DOCKER_QUOTAS_ERROR_MSG= "create {volume_name}: quota size requested but no quota support"``